### PR TITLE
Migrated to Swift 5.0

### PR DESCRIPTION
**Pull request by shahbaz-saleem**
We have migrated project to latest swift 5.0 using xCode 10.2.1(Current Latest), project is now compatible with swift 5.0

Change Set
• Migrated to 5.0 from 4.2
• Updated project information including  …
      – Swift Version to 5.0 from 4.2
      – Pod version to 2.0.3 to 2.4.0

### CODE REVIEW

File: NKVPhonePicker.podspec [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'NKVPhonePicker'
-  s.version          = '2.0.3'
+  s.version          = '2.4.0'
```
[the remark this answers is not in the document]
Comment (shahbaz-saleem) [reply NKVPhonePicker.podspec]: Oh my bad, I wanted to update to 2.0.4 basically, i was a typo.
Both will work. We can go for 2.1.0 or 2.0.4 both.

[the remark this answers is not in the document]
Comment (shahbaz-saleem) [reply NKVPhonePicker.podspec]: Should i send another request after fixing this, or you will do it.

[the remark this answers is not in the document]
Comment (shahbaz-saleem) [reply NKVPhonePicker.podspec]: Hello @NikKovIos , any update on this?

[the remark this answers is not in the document]
Comment (shahbaz-saleem) [reply NKVPhonePicker.podspec]: please update the description too
from "An UITextField subclass to simplify country code's picking. Swift 4.2" to "An UITextField subclass to simplify country code's picking. Swift 5.0"

[the remark this answers is not in the document]
Comment (shahbaz-saleem) [reply NKVPhonePicker.podspec]: Also on cocoapods page, please
https://cocoapods.org/pods/NKVPhonePicker

[the remark this answers is not in the document]
Comment (NikKovIos) [reply NKVPhonePicker.podspec]: Done!

**Pull request merged**